### PR TITLE
Feature/engage ui style

### PR DESCRIPTION
**Pull request by dheeraja00**
TP: 37588, 37777, 37796

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -1004,7 +1004,7 @@ class App extends React.Component<{}, State> {
             active={this.state.outterDrawer}
             activeContentId={this.state.outterDrawerId}
             mode="slide"
-            componentWidth="large"
+            componentWidth="530px"
             componentLabel={this.state.outterDrawerLabel}
```
Comment (ankitsaraogi) [inline example/src/App.tsx]: Can we make '530'as a constant, and use it that way.

File: src/components/Drawer/Drawer.tsx [hunk 2]
```diff
@@ -176,7 +192,7 @@ class Drawer extends React.PureComponent<Props, never> {
   }
 
   renderLayer() {
-    const { active, mode, componentLabel, componentWidth, theme } = this.props;
+    const { active, mode, componentLabel, componentWidth, theme,  } = this.props;
     const containerClassName = this.getContainerClassName();
```
Comment (ankitsaraogi) [inline src/components/Drawer/Drawer.tsx]: Is the extra comma intentional ?

File: src/components/Select/Select.tsx [hunk 3]
```diff
@@ -120,7 +120,7 @@ const select = ({
           id={componentId}
           name={name ? name : 'select'}
           value={value}
-          className={theme.input}
+          className={theme.select}
           disabled={disabled}
```
Comment (ankitsaraogi) [inline src/components/Select/Select.tsx]: Are we supporting 2 kind of 'select', I see one as 'Select' & another one as 'select'.

File: example/src/App.tsx (hunk 1 above)
Comment (dheeraja00) [reply example/src/App.tsx]: @ankitsaraogi user can pass value in px or em or rem, that's why I have kept it like that

File: src/components/Drawer/Drawer.tsx (hunk 2 above)
Comment (dheeraja00) [reply src/components/Drawer/Drawer.tsx]: @ankitsaraogi its in tsconfig that every object should end with comma, that's why its there

File: src/components/Select/Select.tsx (hunk 3 above)
Comment (dheeraja00) [reply src/components/Select/Select.tsx]: @ankitsaraogi yes that's kinda confusing, one style is in delicious theme & other is in component folder, so I am not changing any of them as that can cause any conflict

**Pull request merged**